Cargo.lock

```diff
@@ -12766,7 +12766,6 @@ dependencies = [
  "frame-system-rpc-runtime-api",
  "frame-try-runtime",
  "hex-literal",
- "kusama-runtime",
```

Comment:
[nitpick] `name = "kusama-runtime"` can still be found in Cargo.lock

Reply:
It will be cleaned up automatically during the next codebase upgrade.